config/routes/api.rb

```diff
@@ -8,7 +8,10 @@
 
     constraints FeatureToggleConstraint.new(:dfc_provider) do
       # Mount DFC API endpoints
-      mount DfcProvider::Engine, at: '/dfc-v1.6/'
+      #
+      # The DFC API version depends on the version of the
+      # datafoodconsortium-connector gem.
+      mount DfcProvider::Engine, at: '/dfc-v1.7/'
```

Comment:
I notice the gem version in the lockfile  is different:
```
datafoodconsortium-connector (1.0.0.pre.alpha.6)
```

Also I can't see any `7` in the codebase: https://github.com/datafoodconsortium/connector-ruby

My first thought was "can we import the version number from the gem?" Maybe it shouldn't be automatic though.

Reply:
And on a related note, I wonder if/when to update `DfcProvider::VERSION`. Perhaps it could reflect the version of dfc connector that is supported.

Reply:
The connector version and the DFC version are not "connected". :wink:

DFC v1.7 is just the latest and the Connector was built on that. I don't think that the update should be automatic at this point. There's no official process around DFC version upgrades yet. We will see with future versions if they will need database migrations on other nodes and if we will need to support multiple versions which is not possible with the current Connector (yet).

DfcProvider::VERSION is currently unused like the other engine versions, too. If we ever moved this to a gem then this version would become important but we can ignore it for now. Less complexity.